csv/schema/full_import.csvs

```diff
@@ -79,3 +79,5 @@ custom_cut4: length(*, 255) @optional
 custom_cut5: length(*, 255) @optional
 // Indicates whether the user should be accessing by SSO or not
 sso: length(*, 255) @optional
+// One time password, provided by employer, which later has to be verified by employee
+otp: regex("^[a-zA-Z0-9]*$") @optional
```

Comment:
There's a constraint missing for max length of 32 (or the regex should be changed).